examples/with-typescript/src/App.tsx

```diff
@@ -1,4 +1,4 @@
-import * as React from 'react';
```

Comment:
Put this back?

Reply:
Why use `import * as React` when we have `esModuleInterop`?

Reply:
You right sorry. I got confused for a sec